features/networking/service.feature

```diff
@@ -109,7 +109,8 @@ Feature: Service related networking scenarios
   # @case_id OCP-23895
   @admin
   Scenario: User cannot access the MCS by creating a LoadBalancer service that points to the MCS
-    Given evaluation of `env.master_hosts.first.local_ip` is stored in the :master_ip clipboard
+    Given I store the masters in the :masters clipboard
```

Comment:
```env.master_hosts.first.local_ip``` won't work anymore guess due to now ens is just taking lb. Fixed it with workaround